### PR TITLE
Add startpage for language de

### PR DESCRIPTION
**Pull request by zonky2**

### CODE REVIEW

File: index/de/index.html [hunk 1]
```diff
@@ -0,0 +1,140 @@
+<!DOCTYPE html>
+<html lang="de">
+
+<head>
+    <meta http-equiv="content-type" content="text/html; charset=UTF-8">
+    <meta charset="utf-8">
+    <title>Contao CMS Dokumentation</title>
+    <!-- base href="https://docs.contao.org/" -->
+    <meta name="robots" content="index,follow">
+    <meta name="description" content="Offizielle Contao Dokumentation für Benutzer und Entwickler">
```
Comment (netzarbeiter) [inline index/de/index.html]:
```suggestion
    <meta name="description" content="Offizielle Contao Dokumentation für Benutzer:innen und Entwickler:innen">
```

File: index/de/index.html [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+                <a href="https://contao.org/" id="logo"><img src="../static/contao-logo-corporate.svg" width="200"
+                        height="65"></a> </div>
+        </header>
+        <div id="container">
+            <div id="main">
+                <div class="inside">
+                    <div class="mod_article first block" id="intro">
+                        <div class="ce_text first last block">
+                            <h1>Offizielle Contao Dokumentation</h1>
+                            <p>Die Contao-Dokumentation ist eine Sammlung von Ressourcen, die Ihnen den Start mit
```
Comment (netzarbeiter) [inline index/de/index.html]:
```suggestion
                            <p>Die Contao-Dokumentation ist eine Sammlung von Ressourcen, die dir den Start mit
```

File: index/de/index.html [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+                                Contao erleichtern soll.<br>Alle Unterlagen wurden von der Contao-Community erstellt -<br>
+                                wenn Sie etwas verbessern oder ergänzen möchten, können Sie das gern auf
```
Comment (netzarbeiter) [inline index/de/index.html]:
```suggestion
                                wenn du etwas verbessern oder ergänzen möchtest, kannst du das gern auf
```

File: index/de/index.html [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+                            <div class="content-slider" data-config="0,300,0,">
+                                <div class="slider-wrapper">
+                                    <div class="ce_book current block">
+                                        <figure class="image_container float_above"
+                                            style="margin-top:32px;margin-bottom:16px;margin-left:12px;"
+                                            onclick="window.open('manual/', '_blank')">
+                                            <img src="../static/xicon-manual-c8aecf9e.png" alt="" width="100" height="100">
+                                        </figure>
+                                        <div class="content" onclick="window.open('manual/', '_blank')">
+                                            <h3>Benutzer&shy;handbuch</h3>
```
Comment (netzarbeiter) [inline index/de/index.html]:
```suggestion
                                            <h3>Benutzer:innen&shy;handbuch</h3>
```

File: index/de/index.html [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+                                            <p>Das offizielle Benutzer&shy;handbuch für Contao.</p>
```
Comment (netzarbeiter) [inline index/de/index.html]:
```suggestion
                                            <p>Das offizielle Benutzer:innen &shy;handbuch für Contao.</p>
```

File: index/de/index.html [hunk 6]
```diff
@@ -0,0 +1,140 @@
...
+                                    </div>
+                                    <div class="ce_book current block">
+                                        <figure class="image_container float_above"
+                                            style="margin-top:36px;margin-bottom:16px;margin-left:12px;"
+                                            onclick="window.open('dev/', '_blank')">
+                                            <img src="../static/xicon-developer-d666676a.png" alt="" width="100"
+                                                height="100">
+                                        </figure>
+                                        <div class="content" onclick="window.open('dev/', '_blank')">
+                                            <h3>Entwickler&shy;dokumentation</h3>
```
Comment (netzarbeiter) [inline index/de/index.html]:
```suggestion
                                            <h3>Entwickler:innen&shy;dokumentation</h3>
```

File: index/de/index.html [hunk 7]
```diff
@@ -0,0 +1,140 @@
...
+                                            <p>Aktuelle Doku&shy;mentation für Entwickler für Contao.</p>
```
Comment (netzarbeiter) [inline index/de/index.html]:
```suggestion
                                            <p>Aktuelle Doku&shy;mentation für Entwickler:Innen für Contao.</p>
```

File: index/de/index.html [hunk 8]
```diff
@@ -0,0 +1,140 @@
...
+                                    <div class="ce_book current block">
+                                        <figure class="image_container float_above"
+                                            style="margin-top:32px;margin-bottom:16px;margin-left:12px;"
+                                            onclick="window.open('https://association.contao.org', '_blank')">
+                                            <img src="../static/icon-community.svg" alt="" width="100" height="100">
+                                        </figure>
+                                        <div class="content"
+                                            onclick="window.open('https://association.contao.org', '_blank')">
+                                            <h3>Contao Association</h3>
+                                            <p>Werden Sie Mitglied und geben Sie Ihrem CMS etwas zurück.</p>
```
Comment (netzarbeiter) [inline index/de/index.html]:
```suggestion
                                            <p>Werde Mitglied und geben deinem CMS etwas zurück.</p>
```

File: index/de/index.html [hunk 9]
```diff
@@ -0,0 +1,140 @@
...
+                                            <img src="../static/icon-contao-partner-059781ab.svg" alt="" width="100"
+                                                height="100">
+                                        </figure>
+                                        <div class="content"
+                                            onclick="window.open('https://contao.org/en/support.html', '_blank')">
+                                            <h3>Contao Support</h3>
+                                            <p>Support-Optionen auf der offiziellen Contao-Website.</p>
+                                        </div>
+                                        <a href="https://contao.org/en/support.html" class="read"
+                                            title="Besuchen Sie die Support-Seite auf contao.org" target="_blank">Öffnen</a>
```
Comment (netzarbeiter) [inline index/de/index.html]:
```suggestion
                                            title="Besuche die Support-Seite auf contao.org" target="_blank">Öffnen</a>
```

File: index/de/index.html [hunk 10]
```diff
@@ -0,0 +1,140 @@
...
+                                            onclick="window.open('https://association.contao.org', '_blank')">
+                                            <img src="../static/icon-community.svg" alt="" width="100" height="100">
+                                        </figure>
+                                        <div class="content"
+                                            onclick="window.open('https://association.contao.org', '_blank')">
+                                            <h3>Contao Association</h3>
+                                            <p>Werden Sie Mitglied und geben Sie Ihrem CMS etwas zurück.</p>
+                                        </div>
+                                        <a href="https://association.contao.org/" class="read"
+                                            title="Besuchen Sie die Website der Contao Association." target="_blank">Öffnen</a>
```
Comment (netzarbeiter) [inline index/de/index.html:113]:
```suggestion
                                            title="Besuche die Website der Contao Association." target="_blank">Öffnen</a>
```

**Review by netzarbeiter — CHANGES_REQUESTED** [review]
Thank you, Ingolf

**Review by fritzmg — APPROVED** [review]

**Review by netzarbeiter — APPROVED** [review]

**Comment by fritzmg** [comment]
Deployed: https://docs.contao.org/

**Pull request merged**